roles/manifest/tasks/upload_manifest.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: Upload Subscription Manifest to Foreman
+  theforeman.foreman.subscription_manifest:
+    username: "{{ foreman_username }}"
+    password: "{{ foreman_password }}"
+    server_url: "{{ foreman_server_url }}"
+    organization: "{{ foreman_organization }}"
```

Comment:
have a look at the discussion in https://github.com/theforeman/foreman-ansible-modules/pull/958 around naming common variables. we should agree on something common ;)